podvm/files/etc/systemd/system/confidential-data-hub.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
When I manually start the CDH it fails as follows:

```bash
Sep 15 02:42:24 podvm-nginx-secret-pod-eb58589a systemd[1]: Started Confidential Data Hub Service.
Sep 15 02:42:25 podvm-nginx-secret-pod-eb58589a confidential-data-hub[1381]: Error: init Hub failed: kbs client creation failed: Kbs client error: no `agent.aa_kbc_params` provided in kernel commandline!
Sep 15 02:42:25 podvm-nginx-secret-pod-eb58589a systemd[1]: confidential-data-hub.service: Main process exited, code=exited, status=1/FAILURE
Sep 15 02:42:25 podvm-nginx-secret-pod-eb58589a systemd[1]: confidential-data-hub.service: Failed with result 'exit-code'.
Sep 15 02:42:25 podvm-nginx-secret-pod-eb58589a systemd[1]: confidential-data-hub.service: Scheduled restart job, restart counter is at 1.
Sep 15 02:42:25 podvm-nginx-secret-pod-eb58589a systemd[1]: Stopped Confidential Data Hub Service.
```

Reply:
Hey Suraj, this is due the the cdh not supporting agent-config.toml as mentioned in https://github.com/confidential-containers/cloud-api-adaptor/issues/1424#issuecomment-1719581708